src/screens/RecievedComplaint.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ScrollView, View, StyleSheet } from 'react-native';
+import ComplaintBox from '../complaints/ComplaintBox'; // Adjust the path based on your project structure
+import Navbar from '../navbar/Navbar';
+import ComplaintType from '../complaints/ComplaintType';
+import ClosedComplaint from '../complaints/ClosedComplaint';
+import InProgressComplaint from '../complaints/InProgressComplaint';
+import InitiatedComplaint from '../complaints/InitiatedComplaint';
+
+const RecievedComplaint = () => {
+  const complaintDataArray = [
+    { imageSource: require('../../assets/images/leaves.jpg'), complaintName: 'Complaint 1', otherInfo: 'Information of dummy text how about you now how are you ' },
+    { imageSource: require('../../assets/images/leaves.jpg'), complaintName: 'Complaint 1', otherInfo: 'Information of' },
+    // { imageSource: require('../../assets/images/leaves.jpg'), complaintName: 'Complaint 1', otherInfo: 'Information of' },
+    // { imageSource: require('../../assets/images/leaves.jpg'), complaintName: 'Complaint 1', otherInfo: 'Information of' },
+    // ... more data
+  ];
+
+  return (
+    <ScrollView>
+        <Navbar />
+        <ComplaintType navigation={navigation} />
+      <View style={styles.container}>
+        {complaintDataArray.map((ComplaintData, index) => (
+        //   <ComplaintBox key={index} ComplaintData={ComplaintData} />
+        //   <ClosedComplaint key={index} ComplaintData={ComplaintData}/>
+        // <InProgressComplaint key={index} ComplaintData={ComplaintData}/>
+        <InitiatedComplaint key={index} ComplaintData={ComplaintData} />
+        ))}
+      </View>
+    </ScrollView>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+});
+
+export default RecievedComplaint;
```

Comment:
I'd like to propose a structural refinement for screens within the **complaints** directory:

- If a file represents a full-fledged screen, I recommend relocating it to a dedicated screens directory within the complaints folder. This aligns with best practices for organizing screens and enhances code clarity.

- If a file contains reusable components specific to the complaints functionality, consider moving it to a **components** directory within complaints. This promotes modularity and reusability.

By adhering to these structural guidelines, we can foster a more organized and maintainable codebase.

Reply:
Hmm this is a full-fledged screen and it contain reusable component as well as  it has two main screen i.e
1) Pending Complaint
2) Recieved Complaint
and further individual complaint status 
i got your feedback and soon i will arrange the file like your suggestion

Reply:
@Zakarya525 plz review the structure now it is updated